src/java.management/share/classes/javax/management/ObjectName.java

```diff
@@ -37,6 +37,8 @@
 import java.util.Hashtable;
 import java.util.Map;
 
+import static sun.management.Util.newObjectName;
```

Comment:
What is the motivation for doing the static import? I prefer seeing Util.newObjectName().

Reply:
Right, there are two Util classes, and some classes use both...
e.g. ObjectName.java uses com.sun.jmx.mbeanserver.Util (for Util.wildmatch()), 
and I want to use newObjectName from sun.management.Util.

Reply:
> Right, there are two Util classes, and some classes use both... e.g. ObjectName.java uses com.sun.jmx.mbeanserver.Util (for Util.wildmatch()), and I want to use newObjectName from sun.management.Util.

In that case I'd probably just use the fully qualified class name when making the call and not even bother with an import. Static imports make the code harder to read because it looks like you are calling a method of the class that the call is being made from.

Reply:
Sure, updated to use the method with its qualified class name.